conf/battlegrounds.conf

```diff
@@ -14,7 +14,7 @@
 //= arenas: ({
 //=		//- allowedTypes defines what kind of applications the arena will accept, setting is not case-sensitive and is ok with whitespaces
 //= 		allowedTypes: "Solo | Party" //Arena Accepts solo and party-type joins
-//=     	allowedTypes: "guild|party" //Arena Accepts solo and guild-type joins
+//=     allowedTypes: "guild | party" //Arena Accepts solo and guild-type joins
```

Comment:
look like here you replaces tabs to spaces

Reply:
I think it is OK now.

Please check atcommand_mobinfo_type in gm.conf!